packages/runtime/runtime-definitions/package.json

```diff
@@ -113,9 +114,11 @@
 				"forwardCompat": false,
 				"backCompat": false
 			},
-			"RemovedInterfaceDeclaration_IFluidDataStoreContextEvents": {
-				"forwardCompat": false,
+			"InterfaceDeclaration_ITelemetryContext": {
 				"backCompat": false
+			},
+			"RemovedInterfaceDeclaration_IFluidDataStoreContextEvents": {
+				"forwardCompat": false
```

Comment:
Here we should still restore the `"backCompat": false` that IFluidDataStoreContextEvents` already had

Reply:
Another mistake. Thx!!